### ISSUE TITLE
Cannot load bin/kuby in v0.20.0

### ISSUE THREAD

**Issue by jessie19** [getkuby/kuby-core#131]
## Issue
When we run the
```shell
bundle exec kuby -e sprint remote shell
```

We got this error
```ruby
/Users/jessielin/.rbenv/versions/3.0.6/lib/ruby/gems/3.0.0/gems/kuby-core-0.20.0/bin/kuby:5:in `require': cannot load such file -- sorbet-runtime (LoadError)
```

Then looking the source code in the below line be added in v0.20.0
https://github.com/getkuby/kuby-core/blob/707ba9efacbe34b97310df24016f72c6505d36b4/bin/kuby#L5

However, gem `sorbet-runtime` is only installed for development and test environments, so it is not accessible in `bin/kuby`
https://github.com/getkuby/kuby-core/blob/707ba9efacbe34b97310df24016f72c6505d36b4/Gemfile#L5-L14

## Version diff between 0.18.0 to v.20.0
https://github.com/getkuby/kuby-core/compare/v0.18.0...v0.20.0#diff-738aef34f20cf99931030d782d9df7e556018e7d850e38703e52d761be00ee9b

**Comment by camertron** [comment]
🤦 thanks for the bug report @jessie19, I added this `require` in for debugging purposes and forgot to remove it. I just published v0.20.1 with the fix.

**Comment by jessie19** [comment]
> 🤦 thanks for the bug report @jessie19, I added this `require` in for debugging purposes and forgot to remove it. I just published v0.20.1 with the fix.

Thank you for the quick response!
